### PR TITLE
feat: added navbar simple and with hamburger icon

### PR DESCRIPTION
**Pull request by Rohitprasad83**
# simple navbar
chrome_TzMui6Y2Rd
# navbar with hamburger
chrome_xmbbmrSYXL

### CODE REVIEW

File: navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    margin-right: 0;
+  }
+  .navbar__list__items {
+    font-size: 1.4rem;
+  }
+}
+
+@media only screen and (min-width: 768px){
+  .hamburger {
+    display: none !important;
```
Comment (Naveenchandar) [inline navigation/navigation.css:93]: Please avoid using **!important**

File: navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+@media only screen and (max-width: 768px) {
+  .hamburger {
+    display: flex;
+  }
+  .hamburger__list {
+    width: 18rem;
+    height: auto;
+    border-right: 1px solid var(--dark);
+    border-bottom: 1px solid var(--dark);
```
Comment (Naveenchandar) [inline navigation/navigation.css]: Please use shorthand property for border

File: navigation/navigation.css [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+.navbar__list {
+  list-style: none;
+  max-width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 1rem;
+  gap: 2rem;
+  padding: 1rem;
+  margin-left: auto;
+  margin-right: 5rem;
```
Comment (Naveenchandar) [inline navigation/navigation.css]: Please use shorthand property

File: navigation/navigation.css [hunk 4]
```diff
@@ -0,0 +1,104 @@
+@import url(../utilities/reset.css);
+@import url(../text\ utilities/text.css);
```
Comment (Naveenchandar) [inline navigation/navigation.css:2]: Does this import works ?

File: navigation/navigation.css (hunk 1 above)
Comment (Rohitprasad83) [reply navigation/navigation.css:93]: the css display property is not getting applied if we have a js file overwriting it.

File: navigation/navigation.css (hunk 2 above)
Comment (Rohitprasad83) [reply navigation/navigation.css]: Could you please help me with this, I want to apply border only at right and bottom.

File: navigation/navigation.css (hunk 4 above)
Comment (Rohitprasad83) [reply navigation/navigation.css:2]: yes

File: navigation/navigation.css (hunk 2 above)
Comment (Naveenchandar) [reply navigation/navigation.css]:
```
border: solid green; 
border-width: 2px 0 2px 2px; /* top, right, bottom, left */
```

Comment (Rohitprasad83) [reply navigation/navigation.css]: Thanks, for helping out, I have changed the properties for border and margin, could you please review it again

**Review by prakashsakari — APPROVED** [review]
Looks good to me

**Review by Naveenchandar — APPROVED** [review]
Looks good

**Pull request merged**